forward_engineering/helpers/udtHelper.js

```diff
@@ -44,8 +44,6 @@ const isUdtUsed = type => {
 const isDefinitionTypeValidForAvroDefinition = definition => {
 	if (_.isString(definition)) {
 		return isNamedType(definition);
-	} else if (_.isArray(definition)) {
```

Comment:
according to the specification, it was invalid, because a union is not a named type.